2017/Team-Training/Devspace/index.html

```diff
@@ -0,0 +1,647 @@
+                    <h2>Available by default?</h2>
+                    <ul>
+                    <li>Make similar change to <a href="https://github.com/openmicroscopy/devspace/blob/master/home/jobs/OMERO-web/config.xml" target="_blank">OMERO-web/config.xml</a></li>
+                    <li>Submit a PR</li>
+                    <li>Job Done</li>
+                    </ul>
+                </section>
+</section>
+                <section>
+                    <h2>Access the Docker containers</h2>
+                    <ul>
+                    <li>Log in:<code> ssh omero@devspace_openstack_ip</code></li>
+                    <li>To list the Docker containers, run <code>docker ps</code></li>
+                    </ul>
+<pre><code style="width:1000px">
+CONTAINER ID        IMAGE                                 ... NAMES
+dfb33a9d5de8        devspace_nginx                        ... devspace_nginx_1
+1f15a9e43d4f        devspace_web                          ... devspace_web_1
+e1c08c9ae43b        devspace_testintegration              ... devspace_testintegration_1
+d5faa737a5ce        devspace_omero                        ... devspace_omero_1
+60c351551adc        selenium/node-firefox:3.0.0-dubnium   ... devspace_seleniumfirefox_1
+ff98f484026b        selenium/node-chrome:3.0.0-dubnium    ... devspace_seleniumchrome_1
+63196e8cbf42        nginx:1.10                            ... devspace_nginxjenkins_1
+a568a26fa8f0        devspace_jenkins                      ... devspace_jenkins_1
+96f111f85e5b        postgres                              ... devspace_pg_1
+e292284df351        redis                                 ... devspace_redis_1
+01d82142d674        selenium/hub:3.0.0-dubnium            ... devspace_seleniumhub_1
+</code></pre>
+                </section>
+                <section>
+                    <h2>Access the OMERO.server logs</h2>
+                    <ul>
+                    <li>Access the <code>OMERO-server</code> container
+<pre><code style="width:1000px">
+docker exec -ti devspace_omero_1 bash
+                    </code></pre>
+                    </li>
+                    <li>Access the logs:
+<pre><code style="width:1000px">
+cd /home/omero/workspace/OMERO-server/OMERO.server/
+cd var/log
+                    </code></pre>
+                    </li>
+                    </ul>
+                </section>
+                <section>
+                    <h2>Transfer the OMERO.server logs</h2>
+                    <p align="left">Transfer the logs to your local machine. Two steps process:</p>
+                    <ul>
+                    <li>Copy the logs from the container
+<pre><code style="width:1000px">
+docker cp devspace_omero_1:/home/omero/workspace/OMERO-server/OMERO.server/var/log .
+                    </code></pre>
+                    </li>
+                    <li>Copy the logs to your machine
+<pre><code style="width:1000px">
+scp -r omero@devspace_openstack_ip:/home/omero/log /local/path/for/log
```

Comment:
maybe `scp -Cr` as logs may be large and easily compressed

Reply:
good point, I have tried with small ones only :-)